src/Token/AccessToken.php

```diff
@@ -41,7 +46,11 @@ class AccessToken implements JsonSerializable
     protected $resourceOwnerId;
 
     /**
-     * @param array $options
+     * Constructs an access token.
+     *
+     * @param array $options An array of options returned by the service provider
+     *     in the access token request. The `access_token` option is required.
```

Comment:
I'm really unsure about adding this much documentation to `@param` hints. The code should be the definitive source of what is happening and these kinds of things tend to get out of date and end up causing more harm than good.